### ISSUE TITLE
Incorrect bar height when its value is more than chart's height

### ISSUE THREAD

**Issue by dubroff** [tradingview/lightweight-charts#673]
https://github.com/tradingview/lightweight-charts/blob/5e6327f6acaa157a4a3889dae7f71920fa284696/src/renderers/bars-renderer.ts#L72

I think there is a typo on this line.
You should use **bar.highY** instead of **bar.high**

The current implementation does not work correctly on data series when values are higher than 200.
This can be reprodused using the example code on [JSFiddle](https://jsfiddle.net/TradingView/v0p52anL/) by changing the data in the series to values greater than 200, something like that:

```
barSeries.setData ([
{time: '2018-10-19', open: 280.34, high: 280.99, low: 278.57, close: 279.85},
...
```

**Comment by timocov** [comment]
> I think there is a typo on this line.

It's definitely a typo, nice catch!

Would you like to create a fix for that?

**Comment by dubroff** [comment]
Yes, I will do it.

**Comment by timocov** [comment]
The fix is just published in [3.3.0 version](https://github.com/tradingview/lightweight-charts/releases/tag/3.3.0).

**Comment by ronaldwerd** [comment]
I have a fork of lightweight charts I am using in my react project. I posted something to discord but noticed it was already fixed. Yes nice catch. The rendering glitch was noticeable on XAUDUSD
